.version

```diff
@@ -1 +1 @@
-2.0.0
+2.0.1
```

Comment:
I'd say this was a minor change (2.1.0) or maybe a major change.
I'm ok with leaving it the way it is, since everything is already wired together.

Reply:
Fixed.